Documentation/dev/debugging.md

```diff
@@ -21,10 +21,16 @@ kubectl get pods -n $METERING_NAMESPACE -l app=reporting-operator -o name | cut
 
 The following will open up an interactive presto-cli session where you can interactively query Presto. One thing to note is that this runs in the same container as Presto and launches an additional Java instance, meaning you may run into memory limits for the pod. If this occurs, you should increase the memory request & limits of the Presto pod.
 
+Interacting with presto-cli on macOS:
 ```
 kubectl get pods -n $METERING_NAMESPACE -l app=presto,presto=coordinator -o name | cut -d/ -f2 | xargs -o -I{} kubectl -n $METERING_NAMESPACE exec -it {} -- /usr/local/bin/presto-cli --server localhost:8080 --catalog hive --schema default --user root
 ```
 
+Interacting with presto-cli on Linux:
+```
+kubectl -n $METERING_NAMESPACE exec -it "$(kubectl get pods -l app=presto,presto=coordinator -o name | cut -d/ -f2)"  -- /usr/local/bin/presto-cli --server localhost:8080 --catalog hive --schema default --user root
```

Comment:
We can just use the subprocess option for both mac/linux in all spots. No need to have a different command for each.

Reply:
The inner subshell is missing the ` -n $METERING_NAMESPACE` flag